test/unit/ui.js

```diff
@@ -87,38 +132,206 @@ describe('ui class', () => {
       initStub.restore();
     });
 
-    it('creates new component of type', () => {
+    it('creates new component of type with tracker attached', () => {
       return ui.createComponent('product', productConfig).then(() => {
         assert.equal(1, ui.components.product.length);
+        assert.calledOnce(initStub);
+        assert.calledOnce(trackStub);
         ui.destroyComponent('product', ui.components.product[0].model.id);
       });
     });
 
     it('passes config to constructor', () => {
```

Comment:
What is this testing? The title doesn't really seem related to what's going on inside the test 😕